### ISSUE TITLE
Blacklist feature

### ISSUE THREAD

**Issue by FriedrichFroebel** [FriedrichFroebel/cmanager#9]
Allow the user to move selected (GC or OC) caches to a blacklist which will be ignored when syncing the logs.

**Comment by Lineflyer** [comment]
First of all: Thank you for continuing development for OKAPI sync.

Reading this issue, I have a question about the working mode of cmanager:
If the Sync finds false positives, is it already possible to ignore them permanently? This sounds a bit like this feature request, but I thought I might just not have found the matching option.

At this moment I keep having 4 false positives showing up on each sync and I did not find the "button" to drop them.

**Comment by FriedrichFroebel** [comment]
No, as far as I remember there is no way to permanently ignore false positives for now. This is what this issue/feature request is for - on the one hand ignoring false positives, and on the other hand ignoring caches for other reasons (for example when the OC owner decided to delete the log for whatever reason).

**Comment by Lineflyer** [comment]
Appreciated
